Dockerfile

```
@@ -0,0 +1,9 @@
+FROM python:3.9
+
+RUN pip install fastapi uvicorn
+
+EXPOSE 80
+
+COPY ./app /app
+
+CMD ["uvicorn", "app.RESTful_API:app", "--host", "0.0.0.0", "--port", "80"]
```

With the changes requested in my above comment, this would change to:
```suggestion
CMD ["uvicorn", "agent_testing.RESTful_API:app", "--host", "0.0.0.0", "--port", "80"]
```